puppet-lint-resource_reference_syntax.gemspec

```diff
@@ -22,6 +22,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rspec-its', '~> 1.0'
   spec.add_development_dependency 'rspec-collection_matchers', '~> 1.0'
-  spec.add_development_dependency 'rake', '~> 0'
```

Comment:
Why is this needed? `~> 0` should allow any version.

Reply:
I guess it conflicts with the rake dependency from the changelog generator:
```
bastelfreak ~/puppet-lint-resource_reference_syntax $ bundle install --path .vendor/
[DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set path '.vendor/'`, and stop using this flag
Fetching gem metadata from https://rubygems.org/........
You have requested:
  rake ~> 0

The bundle currently has rake locked at 13.0.1.
Try running `bundle update rake`

If you are updating multiple gems in your Gemfile at once,
try passing them all to `bundle update`
bastelfreak ~/puppet-lint-resource_reference_syntax $ bundle update rake
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Bundler could not find compatible versions for gem "rake":
  In Gemfile:
    rake (~> 0)

    github_changelog_generator was resolved to 1.15.1, which depends on
      rake (>= 10.0)
bastelfreak ~/puppet-lint-resource_reference_syntax $ 
```